styles/light/_light-theme.scss

```diff
@@ -643,7 +643,20 @@ pre code {
   #wallet-xumm {
     content: url("../img/wallets/lightmode/xumm.svg");
   }
+  #wallet-bitfrost {
+    content: url("../img/wallets/lightmode/bitfrost.png");
+  }
+  #wallet-towo {
+    content: url("../img/wallets/lightmode/towo.svg");
+  }
+}
+#wallet-keystone {
+  content: url("../img/wallets/lightmode/keystone.svg");
 }
```

Comment:
The `#wallet-keystone` section needs to be moved _into_ the `#wallets` section above so that it has the proper priority to override the non-light-mode image.